internal/wclayer/wclayer.go

```diff
@@ -1,5 +1,7 @@
 package wclayer
 
+// This pacakge handles the layer creation on windows when importing a container image
```

Comment:
package* 😋 And this package does a little more than that. It provides bindings to the legacy HCS layer management API and provides a higher level interface around these calls to be used for everything to do with layers.

Reply:
Updated. Let me know if the comment still isn't good.

Reply:
Looks good to me :)